### PR TITLE
doc: Document building with west in the Getting Started guide

### PR DESCRIPTION
**Pull request by carlescufi**

### CODE REVIEW

**Review by nashif — APPROVED** [review]

**Comment by carlescufi** [comment]
@dbkinder @nashif @mbolivar this only adds west instructions for building and flashing in the Getting Started page. We may want to consider adding them everywhere instead, but perhaps inverting the order in which they appear. Today west is first:

-----------------
Using west
```
west build -b ...
```
Using cmake and ninja:
```
cmake -GNinja ...
ninja
```
-----------------

we could invert it so:

-----------------
Using cmake and ninja:
```
cmake -GNinja ...
ninja
```
Using west
```
west build -b ...
```
-----------------

File: doc/getting_started/index.rst [hunk 1]
```diff
@@ -222,26 +223,33 @@ formats, called `generators`_. Zephyr supports the following generators:
 
 This documentation and Zephyr's continuous integration system mainly use
 ``Ninja``, but you should be able to use any supported generator to build
-Zephyr applications.
+Zephyr applications, both when using ``cmake`` directly or ``west``.
```
Comment (mbolivar) [inline doc/getting_started/index.rst]:
```suggestion
Zephyr applications, both when using ``cmake`` directly or with ``west``.
```

**Review by mbolivar — APPROVED** [review]
Looks good!

File: doc/getting_started/index.rst (hunk 1 above)
Comment (mbolivar) [reply doc/getting_started/index.rst]: (Just a minor suggestion)

**Review by dbkinder — APPROVED** [review]
LGTM

**Pull request merged**